addons/hr_skills/models/hr_resume_line.py

```diff
@@ -1,25 +1,62 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+import re
+from datetime import datetime
+
+from odoo import api, fields, models
 
 
 class HrResumeLine(models.Model):
     _name = 'hr.resume.line'
     _description = "Resume line of an employee"
     _order = "line_type_id, date_end desc, date_start desc"
 
-    employee_id = fields.Many2one('hr.employee', required=True, ondelete='cascade', index=True)
+    employee_id = fields.Many2one('hr.employee', string="Employee", required=True, ondelete='cascade', index=True)
+    avatar_128 = fields.Image(related='employee_id.avatar_128')
+    company_id = fields.Many2one(related='employee_id.company_id')
+    department_id = fields.Many2one(related='employee_id.department_id')
     name = fields.Char(required=True, translate=True)
-    date_start = fields.Date(required=True)
+    date_start = fields.Date(required=True, default=datetime.today())
```

Comment:
I think this intended to be to be dynamic with today's date So
```suggestion
    date_start = fields.Date(required=True, default=lambda self: datetime.today())
```